ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -33,7 +33,7 @@ Item {
     property string contactToRemove: ""
     property string activeChatId: root.rootStore.chatsModelInst.channelView.activeChannel.id
     property bool isBlocked: root.rootStore.contactsModuleInst.model.isContactBlocked(activeChatId)
-    property bool isContact: root.rootStore.contactsModuleInst.model.isAdded(activeChatId)
+    property bool isContact: root.rootStore.isContactAdded(activeChatId)
```

Comment:
I see this is merged into the `base_bc`, but this is not the way of doing things if we want to respect the architecture we set up on the backend.